autoload/sy/repo.vim

```diff
@@ -46,55 +46,26 @@ function! sy#repo#get_diff(vcs, func) abort
   let [cmd, options] = s:initialize_job(a:vcs)
   let options.func = a:func
 
-  " Neovim
   if has('nvim')
     if job_id
       silent! call jobstop(job_id)
     endif
-
-    let [cwd, chdir] = sy#util#chdir()
-    call sy#verbose(['CMD: '. string(cmd), 'CMD DIR:  '. b:sy.info.dir, 'ORIG DIR: '. cwd], a:vcs)
-
-    try
-      execute chdir fnameescape(b:sy.info.dir)
-    catch
-      echohl ErrorMsg
-      echomsg 'signify: Changing directory failed: '. b:sy.info.dir
-      echohl NONE
-      return
-    endtry
     let b:sy_job_id_{a:vcs} = jobstart(cmd, extend(options, {
+          \ 'cwd':       b:sy.info.dir,
```

Comment:
This wasn't in nvim until 0.1.5.  I'm not sure if we have other version checks for nvim.

Reply:
I expect Nvim users to update their editors more often than Vim users. (And all the popular package managers seem to be on at least `v0.3.x`.)

Although the support for `has('nvim-0.1.5')` predates the release of `v0.1.5`, I think we can just keep `has('nvim')` here.